common/src/main/java/io/seata/common/loader/EnhancedServiceLoader.java

```diff
@@ -113,15 +106,8 @@
      * @throws EnhancedServiceNotFoundException the enhanced service not found exception
      */
     public static <S> S load(Class<S> service, String activateName, Object[] args)
```

Comment:
If service is SINGLETON and we call `load` twice with different args, what instance should return? So far we get the first created instance.

Reply:
We can mark as an issue and reslove it later.

Reply:
fine.